### ISSUE TITLE
[BUG] Dashboard is unreachable when its running within Docker container

### ISSUE THREAD

**Issue by michalgce** [jobrunr/jobrunr#25]
**Describe the bug**
When i start my application within docker container im not able to open dashboard. 
It looks like redirect from / to /dashboard works but it results with 404.

**Environment**
I'm using version 0.9.11
I'm running on JRE / JDK 1.8.0

**To Reproduce**
Try to run jobrunr within docker

**Expected behavior**
Dashboard is accessible when its run within docker container

**Additional context**

Edit: 
After spending some time on debugging i notice the problem is placed within StaticFileHttpHandler.java.
When i start my application from my IDE the path 
```            
final Path toServe = rootDir.resolve(requestUri.substring((contextPath + "/").length()));

```
is accessible. When i create jar file for my whole application, then the ZipFilesSystem changed and setup us my application jar name and it need firstly extract jobrunr jar from my application jar, and theres no way to extract jar which is already inside jar.

**Comment by rdehuyss** [comment]
This is strange as I run end-to-end UI tests to JobRunr in a Docker container.

See https://github.com/jobrunr/jobrunr/blob/master/tests/e2e-ui-jackson/src/test/java/org/jobrunr/tests/e2e/CypressJacksonUIE2ETest.java.

Your analysis however gave her some ideas and I'll look into it ASAP.

**Comment by rdehuyss** [comment]
Hi @michalgce , I released v0.9.12 which may fix your issue. (it might take an hour or 2 when it is available in Maven Central)

Can you confirm?

**Comment by michalgce** [comment]
Sure, im still waiting for an update within maven
